packages/core/src/endowmentsToolkit.js

```diff
@@ -136,6 +137,39 @@ function endowmentsToolkit({
     )
   }
 
+  /**
+   * Creates an object populated with all properties of the source, but applies
+   * all the wrapping and writable mapping
+   *
+   * @template {object} T Deep properties specified in the packagePolicy
+   * @param {T} sourceRef - Object from which to copy properties
+   * @param {object} unwrapTo - For getters and setters, when the this-value is
+   *   unwrapFrom, is replaced as unwrapTo
+   * @param {object} unwrapFrom - For getters and setters, the this-value to
```

Comment:
```suggestion
   * @param {object} [unwrapFrom] - For getters and setters, the this-value to
```

[must] TS won't be able to infer this param is optional otherwise

Reply:
but somehow it does when it's being passed directly to a function that treats it as optional. Or at least it's not complaining. Anyway, updated all to be optional

Reply:
iirc the tests in core aren't typechecked